deploy-grafana.sh

```diff
@@ -17,14 +20,14 @@ docker run --detach \
     --memory=1g \
     -p 0.0.0.0:3000:3000 \
     -v ~/sourcegraph-docker/grafana-disk:/var/lib/grafana \
-    -v $(pwd)/grafana:/etc/grafana \
+    -v ${DIR}/grafana:/sg_config_grafana/provisioning/datasources \
     -e GF_AUTH_ANONYMOUS_ENABLED=true \
     -e GF_AUTH_ANONYMOUS_ORG_NAME=Sourcegraph \
     -e GF_AUTH_ANONYMOUS_ORG_ROLE=Editor \
     -e GF_USERS_ALLOW_SIGN_UP='false' \
     -e GF_USERS_AUTO_ASSIGN_ORG='true' \
     -e GF_USERS_AUTO_ASSIGN_ORG_ROLE=Editor \
-    grafana/grafana:6.1.1@sha256:e7a513bf7f33ef9681b2d35a799136e1ce9330f9055f75dfa2101d812946184b
+    sourcegraph/grafana:3.8
```

Comment:
Could you add the SHA for this (`docker inspect sourcegraph/grafana:3.8` will tell you it), to ensure the image doesn't change by accident if anyone e.g. force pushes the docker tag or something.

Reply:
done